client/src/components/review_create/ImageUploader.jsx

```
@@ -0,0 +1,91 @@
+  // const revokeImage = () => {
+  //   if (imageUrl && imageUrl.startsWith('blob:')) {
+  //     URL.revokeObjectURL(imageUrl);
+  //   }
+  // };
+
+  return (
+    <>
+      <div className={styles.flexBox}>
+        <h2>이미지</h2>
+        <div className={styles.flexBox} onClick={deleteFileHandler}>
+          <img src={deleteFileIcon} alt="deleteFile" className={styles.icon} />
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)